osu.Game.Rulesets.Tau/Objects/Drawables/DrawableSliderTick.cs

```diff
@@ -0,0 +1,149 @@
+﻿using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Rulesets.Tau.Configuration;
+using osu.Game.Rulesets.Tau.UI;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Tau.Objects.Drawables
+{
+    public class DrawableSliderTick : DrawableAngledTauHitObject<SliderTick>
+    {
+        public DrawableSlider DrawableSlider => (DrawableSlider)ParentHitObject;
+
+        public Drawable DrawableCircle;
+
+        private readonly BindableFloat size = new(12f);
+
+        public override bool DisplayResult => false;
+
+        public DrawableSliderTick()
+            : this(null)
+        {
+        }
+
+        public DrawableSliderTick(SliderTick hitObject)
+            : base(hitObject)
+        {
+            Name = "Tick track";
+            Anchor = Anchor.Centre;
+            Origin = Anchor.Centre;
+            RelativeSizeAxes = Axes.Both;
+            Size = Vector2.One;
+
+            AddInternal(DrawableCircle = new Container
+            {
+                RelativePositionAxes = Axes.Both,
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Alpha = 0,
+                AlwaysPresent = true,
+                Size = new Vector2(size.Default),
+                Children = new Drawable[]
+                {
+                    new Circle
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        RelativeSizeAxes = Axes.Both
+                    },
+                    new Circle
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Colour = Color4.Black,
+                        RelativeSizeAxes = Axes.Both,
+                        Size = Vector2.Multiply(Size, 0.5f)
+                    }
+                }
+            });
```

Comment:
Ticks should not have any visuals. I believe I discussed that very briefly in Discord.